src/site/stages/build/drupal/graphql/facilities-fragments/healthCareRegionEvents.node.graphql.js

```diff
@@ -40,9 +40,11 @@ function queryFilter(isAll) {
     ${
       isAll
         ? ''
-        : '{ field: "field_date", value: [$today], operator: GREATER_THAN}'
+        : '{ field: "field_featured" value: "1"}, { field: "field_date", value: [$today], operator: GREATER_THAN}'
```

Comment:
does the date matter if we're looking for featured events? i think it makes sense so that past events cannot be featured, but maybe we should triple check